doc/lua_api.txt

```diff
@@ -23,8 +23,25 @@ functionality in the engine and API, and to document it here.
 Programming in Lua
 ------------------
 
-If you have any difficulty in understanding this, please read
-[Programming in Lua](http://www.lua.org/pil/).
+Lua version 5.1 is used. LuaJIT and PUC Lua are supported.
+
+If you have any difficulty in understanding this document, please read
+[Programming in Lua](https://www.lua.org/pil/).
+Also look at the [Lua 5.1 Reference Manual](https://www.lua.org/manual/5.1/).
+
+Changes to the Lua std library
+------------------------------
+
+Due to security reasons, mods are executed in a sandboxed environment (see also
+`minetest.request_insecure_environment()`). Mods therefore do not have access to
+all functions in the std library. To see what is available, iterate over `_G`.
+
+There are also functions that are available, but were changed in their functionality:
```

Comment:
Either state that this list is non-exhaustive or complete this as [documented here](https://github.com/minetest/minetest_docs/blob/master/doc/environment.adoc)